<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admcontrol</class>
 <widget class="QDialog" name="admcontrol">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>625</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>管理员操作</string>
  </property>
  <property name="windowIcon">
   <iconset resource="picture.qrc">
    <normaloff>:/tushu.png</normaloff>:/tushu.png</iconset>
  </property>
  <widget class="QTabWidget" name="addbook">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab {
    background-color: #7f8c8d;      /* 背景色 */
    color: white;                   /* 文本颜色 */
    border: 1px solid #7f8c8d;      /* 边框 */
    border-radius: 5px;             /* 圆角 */
    min-width: 80px;                /* 最小宽度 */
    padding: 8px 16px;              /* 内边距 */
    margin-right: 2px;              /* 标签间距 */
}
QTabBar::tab:selected {
    background-color: #2c3e50;      /* 选中时的背景色 */
    color: #ecf0f1;                 /* 选中时的文本颜色 */
    border-color: #3498db;          /* 选中时的边框色 */
    font-weight: bold;              /* 粗体文本 */
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <attribute name="title">
     <string>添加书籍</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>91</y>
       <width>65</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
background-color: rgb(171, 171, 171);</string>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>391</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QLineEdit" name="name_1">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	background-color:rgba(255,255,255,150);
	border:none;
	border-bottom:1px solid black;}</string>
        </property>
        <property name="placeholderText">
         <string>书名：</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QLineEdit" name="author">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	background-color:rgba(255,255,255,200);
	border:none;
	border-bottom:1px solid black;}</string>
        </property>
        <property name="placeholderText">
         <string>作者：</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QLineEdit" name="publisher">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	background-color:rgba(255,255,255,200);
	border:none;
	border-bottom:1px solid black;}</string>
        </property>
        <property name="placeholderText">
         <string>出版社：</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QLineEdit" name="isbnEdit">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	background-color:rgba(255,255,255,200);
	border:none;
	border-bottom:1px solid black;}</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>ISBN：</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>39</y>
       <width>65</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
selection-color: rgb(170, 170, 255);
background-color: rgb(171, 171, 171);</string>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>143</y>
       <width>65</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
background-color: rgb(171, 171, 171);</string>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>300</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
background-color: rgb(26, 188, 156);</string>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>257</y>
       <width>74</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="resultLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>260</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(236, 236, 236);</string>
     </property>
     <property name="text">
      <string>请选择类别：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>194</y>
       <width>65</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
background-color: rgb(171, 171, 171);</string>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>631</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/bgr1.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>verticalLayoutWidget</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>pushButton_4</zorder>
    <zorder>pushButton</zorder>
    <zorder>comboBox</zorder>
    <zorder>resultLabel</zorder>
    <zorder>pushButton_5</zorder>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>书籍数据</string>
    </attribute>
    <widget class="QLineEdit" name="name_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color: rgb(237, 237, 237);
	background-color:rgba(255,255,255,200);
	border:none;
	border-bottom:1px solid black;}</string>
     </property>
     <property name="placeholderText">
      <string> 请选择搜索目标：</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>621</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(236, 236, 236,50);</string>
     </property>
    </widget>
    <widget class="QComboBox" name="searchBox">
     <property name="geometry">
      <rect>
       <x>188</x>
       <y>12</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>631</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/bgr2.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>name_2</zorder>
    <zorder>tableView_2</zorder>
    <zorder>searchBox</zorder>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>读者状态控制</string>
    </attribute>
    <widget class="QLineEdit" name="readername_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background-color:rgba(255,255,255,150);
	border:none;
	border-bottom:1px solid black;}</string>
     </property>
     <property name="placeholderText">
      <string>解禁读者名：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="readername">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background-color:rgba(255,255,255,150);
	border:none;
	border-bottom:1px solid black;}</string>
     </property>
     <property name="placeholderText">
      <string>封禁读者名：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>100</y>
       <width>65</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
selection-color: rgb(170, 170, 255);
background-color: rgb(171, 171, 171);</string>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>250</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
background-color: rgb(26, 188, 156);</string>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>170</y>
       <width>65</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
selection-color: rgb(170, 170, 255);
background-color: rgb(171, 171, 171);</string>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>631</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/bgr3.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_3</zorder>
    <zorder>readername_2</zorder>
    <zorder>readername</zorder>
    <zorder>pushButton_10</zorder>
    <zorder>pushButton_11</zorder>
    <zorder>pushButton_12</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>读者数据</string>
    </attribute>
    <widget class="QLineEdit" name="name_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color:rgba(255,255,255,200);
	border:none;
	border-bottom:1px solid black;}</string>
     </property>
     <property name="placeholderText">
      <string>请选择搜索目标：</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>621</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(236, 236, 236,50);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>631</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/bgr4.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_4</zorder>
    <zorder>name_3</zorder>
    <zorder>tableView</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>name_1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>author</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>publisher</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>isbnEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
